internal/devicescale/impl_desktop_nonunix.go

```diff
@@ -12,8 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !(dragonfly || freebsd || linux || netbsd || openbsd || solaris) && !android && !ios && !js
+// +build !dragonfly,!freebsd,!linux,!netbsd,!openbsd,!solaris,!android,!ios,!js
```

Comment:
```
//go:build (darwin || linux || windows) && !android && !ios
```

Reply:
Oops, yeah, Darwin. Also adjusted the X11 case too to exclude Darwin.

Reply:
Wrote them like this now to make it obvious that they're correct:

```
impl_desktop.go://go:build !android && !ios && !js
impl_desktop_notx.go://go:build !android && !ios && !js && (darwin || windows)
impl_desktop_x.go://go:build !android && !ios && !js && !(darwin || windows)
```